### ISSUE TITLE
error on blitz install quirrel

### ISSUE THREAD

**Issue by brantchoate** [blitz-js/blitz#1804]
### What is the problem?
blitz install quirrel produces the following error:

```
TSError: ⨯ Unable to compile TypeScript:
    ../../../../tmp/blitz-recipe-blitz-js-blitz/index.ts:
    30:29 - error TS2339: Property 'packageJson' does not
     exist on type '{ document(): string; app(): string; 
    entry(): string; babelConfig(): string; 
    blitzConfig(): string; }'.
```

### Steps to Reproduce

1. Run `blitz install quirrel`

### Versions

```
debug: blitzPkgPath: /home/brant/Sites/bapp/node_modules/blitz/dist/index.js
debug: cliPkgPath: /home/brant/Sites/bapp/node_modules/@blitzjs/cli/lib/src/index.js 

Linux 4.19 | linux-x64 | Node: v12.18.3

blitz: 0.29.5 (global)
blitz: 0.29.5 (local)

  Package manager: yarn 
  System:
    OS: Linux 4.19 Debian GNU/Linux 10 (buster) 10 (buster)
    CPU: (8) x64 Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
    Memory: 22.46 GB / 25.01 GB
    Shell: 5.0.3 - /bin/bash
  Binaries:
    Node: 12.18.3 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/cli: ~2.14 => 2.14.0 
    @prisma/client: ~2.14 => 2.14.0 
    blitz: 0.29.5 => 0.29.5 
    react: 0.0.0-experimental-3310209d0 => 0.0.0-experimental-3310209d0 
    react-dom: 0.0.0-experimental-3310209d0 => 0.0.0-experimental-3310209d0 
    typescript: 4.1.3 => 4.1.3 
```

### Other

Please include applicable logs and screenshots that show your problem.

**Comment by Skn0tt** [comment]
`Property 'packageJson' does not exist on type`

It seems like your Blitz installation is missing this line:

https://github.com/blitz-js/blitz/blob/b41dd822fad8b94f0ae909dcaa6a4f511e03bab9/packages/installer/src/utils/paths.ts#L24

That's unexpected, considering you're on Blitz 0.29.5. @aem do you have an idea what could cause this?

**Comment by flybayer** [comment]
@Skn0tt because the installer package changes haven't been published to npm yet.

I just published those changes in [0.29.7](https://github.com/blitz-js/blitz/releases/tag/v0.29.7), so upgrade to that and it should work.
